js/components/dashboard/expand-students.js

```diff
@@ -0,0 +1,28 @@
+import React, { PureComponent } from 'react'
+import { Map, List } from 'immutable'
+import { SET_STUDENTS_EXPANDED } from '../../actions/dashboard'
+import Button from '../report/button'
```

Comment:
If some component gets reused, I'd move it to `../common/...`, so developer knows that he should check both app types while working on it (e.g. changing styles).

Reply:
Done!